docs/user/dashboard.asciidoc

```diff
@@ -137,26 +152,21 @@ tools. To create a short URL, you must have write access to {kib}.
 === Import and export dashboards
 
 To import and export dashboards, go to *Management > Saved Objects*. For details,
-see <<managing-saved-objects, Managing saved objects>>. 
+see <<managing-saved-objects, Managing saved objects>>.
 
 [float]
 [[viewing-detailed-information]]
 === Inspect and edit elements
```

Comment:
This section feels like it belongs on a separate page.  Also, it needs more information about why you would want to inspect and edit an element. Maybe a separate PR?

Reply:
I'm moving this section up. I inspect/edit elements as I'm arranging them on the dashboard, and I definitely do this before I save the dashboard.